### DISCUSSION TITLE
Edit QTable cell without Popup Edit

### DISCUSSION THREAD

**Discussion by JS-Aibel** [quasarframework/quasar#16179] [General - Components / Directives / etc] [answered]
Hi,

Is it possible to edit a QTable cell just by clicking inside the cell?
This will make the user experience faster than using a popup for each edit.

**Comment by metalsadman** [comment, marked as answer]
yeah use one of the body slots, make the cell hold a borderless qinput. you can simply take this example https://quasar.dev/vue-components/table#example--popup-editing and remove the q-popup-edit leave the qinput.
ie. 1st column
https://codepen.io/metalsadman/pen/mdQZqKy?editors=101
just be careful when you click on sort, if you edit the field while sort was on, then it will trigger sort while you're editing so it will jump.
